File: vuln.js
const express = require('express');
const app = express();
app.use(express.json());

app.post('/login', (req, res) => {
  const q = "SELECT * FROM users WHERE user='" + req.body.user + "'";
  db.query(q, (err, result) => res.json(result));
});

app.listen(3000);

File: vuln.py
import sqlite3
from flask import Flask, request

app = Flask(__name__)
db = sqlite3.connect('users.db', check_same_thread=False)

@app.route('/user')
def user():
    q = "SELECT * FROM users WHERE id=" + request.args['id']
    return db.execute(q).fetchall()
